spring-kafka/src/test/java/org/springframework/kafka/listener/KafkaMessageListenerContainerTests.java

```diff
@@ -87,9 +87,13 @@
 
 	private static String topic7 = "testTopic7";
 
+	private static String topic8 = "testTopic8";
+
+	private static String topic9 = "testTopic9";
```

Comment:
Looks like `topic9` is out of use, but I guess you will need it when you've done with `batchErrorHandling` 😄